website/src/theme/TOC/DocReleaseBadge.tsx

```
@@ -0,0 +1,52 @@
+
+interface DocReleaseData {
+  releaseMap: Record<string, string>;
+}
+
+export default function DocReleaseBadge(): JSX.Element | null {
+  const { pathname } = useLocation();
+
+  // Get the release map from our plugin's global data.
+  let releaseData: DocReleaseData | undefined;
+  try {
+    releaseData = usePluginData('doc-release-data') as DocReleaseData;
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)